manifests/params.pp

```diff
@@ -205,27 +205,4 @@
       fail("Osfamily ${::osfamily} and ${::operatingsystem} is not supported")
     }
   }
-
-  case $::operatingsystem {
-    'Debian': {
-      case $::operatingsystemmajrelease {
-        '8': {
-          $service_provider = pick($service_provider, 'systemd')
```

Comment:
Is there nothing in the code that references $mongodb::params::service_provider anymore?

Reply:
There is, but in globals it defaults to undef anyway.